cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -281,20 +282,41 @@ func main() {
 		return
 	}
 
-	swagger, err := util.LoadSwagger(flag.Arg(0))
+	spec, err := util.LoadOpenAPI(flag.Arg(0))
 	if err != nil {
 		errExit("error loading swagger spec in %s\n: %s\n", flag.Arg(0), err)
```

Comment:
Shouldn't be this?

```suggestion
 		errExit("error loading OpenAPI spec in %s\n: %s\n", flag.Arg(0), err)
```

Reply:
It's unfortunately a wider issue https://github.com/oapi-codegen/oapi-codegen/issues/672